### PR TITLE
Update HurricaneElectric.xml

### PR DESCRIPTION
**Pull request by cschanaj**
#17965

### CODE REVIEW

File: src/chrome/content/rules/HurricaneElectric.xml [hunk 1]
```diff
@@ -3,41 +3,27 @@
...
 	<target host="he.net" />
-	<target host="*.he.net" />
-	<target host="hurricaneelectric.net" />
-
-
-	<securecookie host="^(?:.*\.)?he\.net$" name=".+" />
-
-
-	<rule from="^http://(?:www\.)?h(?:e\.com|urricaneelectric\.net)/"
-		to="https://www.he.net/" />
```
Comment (J0WI) [inline src/chrome/content/rules/HurricaneElectric.xml]: Shouldn't we keep a rule like this?

**Comment by J0WI** [comment]
This will close #17584

File: src/chrome/content/rules/HurricaneElectric.xml (hunk 1 above)
Comment (cschanaj) [reply src/chrome/content/rules/HurricaneElectric.xml]: no, I missed this rule when I rewrote the ruleset. thanks.

File: src/chrome/content/rules/HurricaneElectric.xml [hunk 2]
```diff
@@ -22,8 +22,14 @@
 	<target host="cust.he.net" />
 	<target host="dns.he.net" />
```
Comment (J0WI) [inline src/chrome/content/rules/HurricaneElectric.xml]: `faq.he.net`

File: src/chrome/content/rules/HurricaneElectric.xml [hunk 3]
```diff
@@ -22,8 +22,14 @@
...
 	<target host="ipv6.he.net" />
+	
+	<target host="hurricaneelectric.net" />
+		<test url="http://hurricaneelectric.net/tour/" />
 
 	<securecookie host=".+" name=".+" />
+	
+	<rule from="^http://hurricaneelectric\.net/"
+		  to="https://www.he.net/" />
```
Comment (J0WI) [inline src/chrome/content/rules/HurricaneElectric.xml]: same for `(www\.)?he.com`

**Comment by J0WI** [comment]
Thanks!

**Pull request merged**